app/main.py

```diff
@@ -1 +1,21 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import VaccineError, NotWearingMaskError
+
+
+def go_to_cafe(friends: list[dict], cafe: Cafe) -> str:
+    masks_to_buy = 0
+    try:
+        for friend in friends:
+            try:
+                cafe.visit_cafe(friend)
+            except NotWearingMaskError:
+                masks_to_buy += 1
+            except VaccineError:
+                raise
+    except VaccineError as e:
+        message = ''.join(list(e.args))
+        return message
+    else:
```

Comment:
There's no point in using `else` condition after `return` in preceding `if`statement. You can do it like this:
```python
if ...:
    return ...

return ...

Reply:
Thanks, for the good advice!